docs/state-sync.md

```diff
@@ -0,0 +1,155 @@
+# State Sync
+
+## What is State sync?
+
+The Provenance 1.10.0 release comes with Tenderming Core 0.34 which includes support for state sync. State sync allows a new node to join a network by fetching a snapshot of the application state at a recent height vs. fetching and replaying all historical blocks (which can take days). This can reduce the time needed to sync with the network down to minutes.
+
+<!-- TOC -->
+- [What is State sync?](#what-is-state-sync)
+    - [Starting a new node from State Sync](#starting-a-new-node-from-state-sync)
+        - [Mainnet](#mainnet)
+        - [Testnet](#testnet)
+    - [Power users](#power-users)
+
+
+### Starting a new node from State Sync
+___
+
+#### Mainnet
+
+##### 1. Install the `provenanced` binary
+
+```
+# setup home directory (or directory of your choosing)
+export PIO_HOME=~/.provenanced
+
+# clone and checkout
+git clone https://github.com/provenance-io/provenance.git
+cd provenance
+git checkout tags/v1.10.0 -b v1.10.0
+
+# install binary
+make clean install
+
+# initialize node
+provenanced init choose-a-moniker --chain-id pio-mainnet-1
+curl https://raw.githubusercontent.com/provenance-io/mainnet/main/pio-mainnet-1/genesis.json> genesis.json
+mv genesis.json $PIO_HOME/config
+
+# update config to use 'cleveldb'
+sed -i.orig -E "s|^(db_backend[[:space:]]+=[[:space:]]+).*$|\1\"cleveldb\"| ;" $PIO_HOME/config/config.toml
+```
+
+##### 2. Start a new node from State Sync
+
+```  
+# setup sync node
+PIO_RPC=`host rpc-0.provenance.io | awk '{print $4}'`:26657
```

Comment:
We should add a comment about rpc-1 and rpc-2 as well... I always worry that we are going to have the rpc-0 node maxed out and cause issues since it is the obvious default for everything.